src/Symfony/Component/Console/Helper/SymfonyQuestionHelper.php

```diff
@@ -29,6 +29,8 @@ class SymfonyQuestionHelper extends QuestionHelper
 {
     /**
      * {@inheritdoc}
+     *
+     * @deprecated since version 3.3, to be removed in 4.0
```

Comment:
Indeed, I think I would talk only about the removed validation. Btw this needs to be documented :)